.github/workflows/tiobe.yaml

```diff
@@ -0,0 +1,39 @@
+name: TIOBE Quality Checks
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 7 1 * *'
+
+jobs:
+  TICS:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+  
+      - name: Install dependencies
+        run: |
+          go install honnef.co/go/tools/cmd/staticcheck@v0.5.1
+          go get github.com/axw/gocov/gocov
+          go get github.com/AlekSi/gocov-xml
```

Comment:
Should we `go install` these and pin them at a specific version too, like the staticcheck one?